besu/src/test/java/org/hyperledger/besu/cli/BesuCommandTest.java

```diff
@@ -2073,6 +2110,31 @@ public void rpcHttpOptionsRequiresServiceToBeEnabled() {
     assertThat(commandErrorOutput.toString(UTF_8)).isEmpty();
   }
 
+  @Test
+  public void rpcHttpOptionsRequiresServiceToBeEnabledToml() throws IOException {
```

Comment:
Since we have junit 5 now, creating a parameterized tests for both command line and toml woud be nice and keep this BesuCOmmandTest a little bit smaller

Reply:
Could start a BesuCommandTomlTest file perhaps?